pwnlib/elf/elf.py

```diff
@@ -1328,13 +1328,23 @@ def relro(self):
         .. _DF_BIND_NOW: http://refspecs.linuxbase.org/elf/gabi4+/ch5.dynamic.html#df_bind_now
 
         """
+        if not any('GNU_RELRO' in str(s.header.p_type) for s in self.segments):
```

Comment:
I checked it by generating multiple really simple binaries with `-Wl,-z,now` but without `-Wl,-z,relro`. This way DF_BIND_NOW and DF_1_NOW were set but it had no GNU_RELRO header.  
I loaded it with gdb and found that the GOT table was filled at start but it was mapped as `rw-p`. 
This case imho should be treated as "No RELRO" (`checksec.sh` also reports "No RELRO"). If I add the `-Wl,-z,-relro` flag then the GOT is mapped `r--p` (as it should be).

Should I upload binaries to check this into `pwnlib/data/elf`?